pkgs/development/libraries/libjwt/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, lib, fetchurl, autoreconfHook, pkg-config, jansson, openssl }:
+
+stdenv.mkDerivation rec {
+  name = "libjwt-${version}";
+  version = "v1.12.1";
+
+  src = fetchurl {
+    url = "https://github.com/benmcollins/libjwt/archive/${version}.tar.gz";
+    sha256 = "05yc8giw1kj5cwqd0a15p7w9wlymkzk11sahcc3hnd1psi8457nj";
+  };
+
+  buildInputs = [ jansson openssl ];
+  nativeBuildInputs = [ autoreconfHook pkg-config ];
+
+  meta = with lib; {
+    homepage = "https://github.com/benmcollins/libjwt";
+    description = "JWT C Library";
+    license = licenses.mpl20;
+    platforms = platforms.all;
```

Comment:
Please add yourself as a maintainer the file https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix as a separate commit with the commit message `maintainers: add pnotequalnp` and add yourself as a maintainer of this library.